### DISCUSSION TITLE
plugin support: reactive.nvim

### DISCUSSION THREAD

**Discussion by zhengpd** [olimorris/onedarkpro.nvim#234] [Ideas]
reactive.nvim is a nice plugin providing more visual feedbacks in neovim. It would be nice if onedarkpro.nvim has builtin support for it like catppuccin does. 

- https://github.com/rasulomaroff/reactive.nvim
- https://github.com/catppuccin/nvim/blob/main/lua/reactive/presets/catppuccin-latte-cursorline.lua

**Comment by olimorris** [comment]
Thanks for this. I can't say I've ever used the plugin but would be totally up for a PR that added support for this. I think with the lighten and darken helpers in the theme you could accomplish something very quick with it.
